package.json

```diff
@@ -21,9 +21,13 @@
     "@mdx-js/react": "^1.6.22",
     "@smallstep/internal": "./vendor/internal",
     "@smallstep/step-ui": "./vendor/step-ui",
+    "@styled-icons/fa-solid": "^10.34.0",
+    "algoliasearch": "^4.13.1",
     "cross-fetch": "^3.1.5",
+    "eventsource": "1.1.1",
```

Comment:
Is this specified somewhere as being needed to explicitly install for the plugin? Can we uninstall it?